pkgs/tools/misc/hexbinhex/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hexbinhex";
+  version = "unstable-2023-03-31";
```

Comment:
Ideally, you'd ask upstream if it was possible to create a release and link the issue here.

Reply:
I have now created an issue in the original github repository (https://github.com/dj-on-github/hexbinhex/issues/2).

Reply:
Upstream released a new version with an accompanying tag. The package is now changed to `version = "1.1"`.